src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs

```diff
@@ -40,7 +40,7 @@ public abstract class FilterActionInvoker : IActionInvoker
 
         public FilterActionInvoker(
             [NotNull] ActionContext actionContext,
-            [NotNull] INestedProviderManager<FilterProviderContext> filterProvider,
+            [NotNull] IFilterProvider[] filterProviders,
```

Comment:
You are missing this is a design pr

-----Original Message-----
From: "Pranav K" <notifications@github.com>
Sent: ‎2/‎22/‎2015 5:33 PM
To: "aspnet/Mvc" <Mvc@noreply.github.com>
Cc: "Yishai Galatzer" <yishai_galatzer@yahoo.com>
Subject: Re: [Mvc] [Design] Remove INestedProvider (#2060)

In src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs:
> @@ -40,7 +40,7 @@ public abstract class FilterActionInvoker : IActionInvoker
>  
>          public FilterActionInvoker(
>              [NotNull] ActionContext actionContext,
> -            [NotNull] INestedProviderManager<FilterProviderContext> filterProvider,
> +            [NotNull] IFilterProvider[] filterProviders,
Maybe use IReadOnlyList here?
—
Reply to this email directly or view it on GitHub.